package.json

```diff
@@ -25,13 +25,14 @@
   },
   "homepage": "https://github.com/CleverTap/clevertap-node",
   "dependencies": {
-    "querystring": "^0.2.0",
-    "request": "^2.69.0"
+    "clevertap": "^1.2.0",
```

Comment:
@KambleSonam 
- why are we adding "clevertap" as a dependency?
- We need to update the version in this file to v1.3.0